huxley/core/models.py

```diff
@@ -444,3 +270,195 @@ def save(self, *args, **kwargs):
     class Meta:
         db_table = u'delegate'
         ordering = ['school']
+
+
+class Registration(models.Model):
+    CONTACT_TYPE_OPTIONS = ((ContactType.FACULTY, 'Faculty'),
+                            (ContactType.STUDENT, 'Student'), )
+
+    GENDER_OPTIONS = ((ContactGender.MALE, 'Male'),
+                      (ContactGender.FEMALE, 'Female'),
+                      (ContactGender.OTHER, 'Other'),
+                      (ContactGender.UNSPECIFIED, 'Unspecified'), )
+
+    registered = models.DateTimeField(auto_now_add=True)
+
+    primary_name = models.CharField(max_length=128)
+    primary_gender = models.PositiveSmallIntegerField(
+        choices=GENDER_OPTIONS, default=ContactGender.UNSPECIFIED)
+    primary_email = models.EmailField()
+    primary_phone = models.CharField(max_length=32)
+    primary_type = models.PositiveSmallIntegerField(
+        choices=CONTACT_TYPE_OPTIONS, default=ContactType.FACULTY)
+    secondary_name = models.CharField(max_length=128, blank=True)
+    secondary_gender = models.PositiveSmallIntegerField(
+        choices=GENDER_OPTIONS, blank=True, default=ContactGender.UNSPECIFIED)
+    secondary_email = models.EmailField(blank=True)
+    secondary_phone = models.CharField(max_length=32, blank=True)
+    secondary_type = models.PositiveSmallIntegerField(
+        choices=CONTACT_TYPE_OPTIONS, blank=True, default=ContactType.FACULTY)
+
+    times_attended = models.PositiveSmallIntegerField(default=0)
+    beginner_delegates = models.PositiveSmallIntegerField()
```

Comment:
`num_beginner_delegates`, ditto for the other four. Maybe a little verbose, but worth it to be explicit? What do you think?